src/lib/tabs/tab-body.ts

```diff
@@ -202,4 +210,29 @@ export class MatTabBody implements OnInit {
         position == 'left-origin-center' ||
         position == 'right-origin-center';
   }
+
+  /** Computes the position state that will be used for the tab-body animation trigger. */
+  private _computePositionAnimationState(dir: Direction = this._getLayoutDirection()) {
```

Comment:
From what I can tell you're never passing a `dir` to the calls of this method. Consider moving it into a variable.

Reply:
A `dir` is being passed from the `subscribe` in the constructor.